scripts/docker/debian.8/Dockerfile

```diff
@@ -7,60 +7,40 @@
 FROM debian:jessie
 
 # Misc Dependencies for build
-RUN apt-get update && apt-get -qqy install curl unzip gettext sudo
+RUN apt-get update && \
+    apt-get -qqy install \
+        curl \
+        unzip \
+        gettext \
+        sudo && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
```

Comment:
Did you try with just `rm -rf`?  Official Dockerfiles usually just call `rm -rf`.  Also another typical practice that can help keep the image size down is to use the `--no-install-recommends` apt-get install.